consensus/types/src/execution_block_hash.rs

```diff
@@ -20,7 +20,7 @@ use std::fmt;
 )]
 #[derivative(Debug = "transparent")]
 #[serde(transparent)]
-pub struct ExecutionBlockHash(Hash256);
+pub struct ExecutionBlockHash(pub Hash256);
```

Comment:
```suggestion
pub struct ExecutionBlockHash(Hash256);
```

It doesn't really matter, but we can avoid touching this file with a simple change (there's one more related suggestion in this review).

Reply:
Nice catch! Fixed in [7554d84](https://github.com/sigp/lighthouse/pull/3807/commits/7554d8492c946a6b41489b15c15fc06c18a0e0d9)